recipes/sherlockml-dataclean/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "sherlockml-dataclean" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "1c3a8b1fe7f83ed796e0c2e3666fc51dd83dc9213fa2c49e3efe26cec2211653" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - jupyter
```

Comment:
Is this a build dependency?

Reply:
@pbugnion I based this bit on your gmaps recipe, is it correct?

Reply:
This recipe runs `jupyter nbextension {enable,disable} ...` in the pre-unlink/post-unlink scripts, so it definitely needs to be there at runtime, but presumably not at build-time?